### ISSUE TITLE
[Feature suggestion] Markdown in text fields and in chat

### ISSUE THREAD

**Issue by GwynethLlewelyn** [lupuslabs/n3qExt#162]
With the current trend of allowing people at least _some_ ability to 'decorate' their plain text with some Markdown, would you consider adding a Markdown processor to those areas as well?

There's a plethora of excellent tools available: https://byby.dev/js-markdown-libs

Of those, I only tried `marked` and `remarkable`,  I think. But even a very simple and rudimentary, lightweight Markdown processor could go a long way...

**Comment by wolfspelz** [comment]
Yes we would. Good idea. Already using MD in our blog client side.
